modules/banner/react/lib/Banner.tsx

```diff
@@ -47,23 +54,30 @@ const BannerWrapper = styled('button')<BannerProps>(
     display: 'flex',
     alignItems: 'center',
     transition: 'background-color 120ms',
-    '&:focus': {
-      outline: 'none',
-      ...focusRing(2, 2),
-    },
     '&:hover': {
       cursor: 'pointer',
     },
   },
-  ({error, variant}) => ({
-    backgroundColor: error === ErrorType.Error ? colors.cinnamon500 : colors.cantaloupe400,
-    color: error === ErrorType.Error ? colors.frenchVanilla100 : colors.blackPepper400,
+  ({error, theme}) => {
+    theme = getTheme(theme); // eslint-disable-line no-param-reassign
+    return {
+      backgroundColor:
+        error === ErrorType.Error ? theme.palette.error.main : theme.palette.alert.main,
+      color: error === ErrorType.Error ? theme.palette.error.contrast : colors.blackPepper400,
+      '&:hover': {
```

Comment:
Note: I realized we don't have active states for our `Banner` component. The color/contrast gets pretty terrible based on our current system, so we should consider this.